src/Transformers/BooleanConverter.php

```diff
@@ -0,0 +1,95 @@
+<?php
+
+namespace Rubix\ML\Transformers;
+
+use Rubix\ML\DataType;
+use Rubix\ML\Exceptions\InvalidArgumentException;
+
+/**
+ * Boolean Converter
+ *
+ * Convert boolean true/false values to continuous or categorical values.
+ *
+ * @category    Machine Learning
+ * @package     Rubix/ML
+ * @author      Zach Vander Velden
+ */
+class BooleanConverter implements Transformer
+{
+    /**
+     * The value used to replace boolean value `true` with. Use a string value to represent the boolean as a categorical
+     * value. Use a float or int to represent the boolean as a continuous value.
+     *
+     * @var string|float|int
+     */
+    protected $trueValue;
+
+    /**
+     * The value used to replace boolean value `false` with. Use a string value to represent the boolean as a categorical
+     * value. Use a float or int to represent the boolean as a continuous value.
+     *
+     * @var string|float|int
+     */
+    protected $falseValue;
+
+    /**
+     * BooleanConverter constructor.
+     * @param mixed $trueValue
+     * @param mixed $falseValue
+     */
+    public function __construct($trueValue = 'true', $falseValue = 'false')
+    {
+        if (!is_float($trueValue) && !is_int($trueValue) && !is_string($trueValue)) {
```

Comment:
Would `is_numeric()` work here in place of the calls to `is_float()` and `is_int()`?